.gitignore

```diff
@@ -6,3 +6,8 @@ artifacts/
 
 # Avoid squashing local people's file for now
 /.vscode
+
+# Python setuptools artifacts
+*.egg-info/
+dist/
+bin/
```

Comment:
This is just stuff that would get created when making the PA wheel. Don't want it accidentally getting added when doing wheel dev.